src/screens/CityList/CityList.tsx

```diff
@@ -0,0 +1,69 @@
+import type { FC } from "react"
+import { FlatList, StyleSheet, Text } from "react-native"
+import { useNavigation } from "@react-navigation/native"
+import type { StaticScreenProps } from "@react-navigation/native"
+import { useCities } from "../../services/restaurant/hook"
+import { Box, Press, Typography } from "../../components"
+
+type Props = StaticScreenProps<{
+  state: string
+}>
+
+const CityList: FC<Props> = ({ route }) => {
+  const { state } = route.params
+  const navigation = useNavigation()
+  const { data: cities, error, isPending } = useCities(state || "")
+
+  if (error) {
+    return (
+      <Box style={styles.container}>
+        <Text>Error: {error.message}</Text>
+      </Box>
+    )
+  }
+
+  if (isPending) {
+    return (
+      <Box style={styles.container}>
+        <Text>Loading...</Text>
+      </Box>
+    )
+  }
+
+  return (
+    <Box padding="m" style={styles.container}>
+      <Typography variant="heading">Select a City: </Typography>
+      <FlatList
+        style={styles.options}
+        data={cities}
+        renderItem={({ item: cityItem }) => (
+          <Press
```

Comment:
Not in this PR, but I would like us to not have buttons in the list views and do something more like the “simple table” here: https://developer.apple.com/documentation/uikit/views_and_controls/table_views

Reply:
Yeah, I agree. The plan with Ryan was to make sure we had all of the functionality in place and then figure out the styling views. That said I'm aware this implementation is a bit problematic. I'll try to get the list views in before we settle on styling the whole app.